subprojects/soak/soak.gradle.kts

```diff
@@ -0,0 +1,38 @@
+import org.gradle.gradlebuild.unittestandcompile.ModuleType
+
+plugins {
+    gradlebuild.classycle
+    `kotlin-library`
+}
+
+dependencies {
+    testFixturesApi(project(":internalIntegTesting"))
+    testImplementation(project(":kotlinDslTestFixtures"))
+}
+
+gradlebuildJava {
+    moduleType = ModuleType.INTERNAL
+}
+
+testFixtures {
+    from(":core")
+}
+
+tasks.matching { it.name in listOf("integTest", "java9IntegTest") }.configureEach {
+    require(this is Test)
```

Comment:
:o: could be replaced by chaining `.withType<Test>()` in the above statement

Reply:
After a conversation with @jjohannes I've removed the reference to `java9IntegTest` then was able to replace the whole thing by configuration via type-safe accessor.